src/gc/gcpriv.h

```diff
@@ -4311,9 +4311,9 @@ dynamic_data* gc_heap::dynamic_data_of (int gen_number)
 // In the code we also rely on the assumption that one card_table entry (uint32_t) covers an entire os page
 //
 #if defined (BIT64)
-#define card_size ((size_t)(2*OS_PAGE_SIZE/card_word_width))
+#define card_size ((size_t)(8192/card_word_width))
 #else
-#define card_size ((size_t)(OS_PAGE_SIZE/card_word_width))
+#define card_size ((size_t)(4096/card_word_width))
 #endif // BIT64
 
```

Comment:
They are empirical, yes, but they are also tied to `GetWriteWatch` operating at page granularity. If we change these, we'll also need to change the software write watch tables and card bundle tables (cc @adityamandaleeka)